case-study-savas-labs.mdx

```diff
@@ -0,0 +1,86 @@
+- **Location:** Remote, HQ in Raleigh NC
+- **Size:** 30-35 employees
+
+[Savas Labs](https://savaslabs.com/), a full-service digital agency headquartered in North Carolina, is dedicated to delivering impactful and purpose-driven digital solutions. With a team of 30-35 professionals, including designers, developers, and project managers, Savas Labs has built a reputation for its focus on philanthropic work, particularly within academia and non-profit sectors. The agency has partnered with esteemed institutions like Duke University and Charlotte University. Their commitment to thorough discovery processes and meaningful client collaboration sets them apart from traditional agencies that often take a more generic approach to digital projects.
+
+## The task ahead
+
+Savas Labs was approached by the [Environmental Integrity Project](https://environmentalintegrity.org/) (EIP) to do a complete teardown and replacement of an existing Angular 1.0 application backed by a Postgres database. The purpose of the site is to show complicated environmental data (CO2e, economic effects, cancer risk, projected deaths, water crossings) of hundreds of facilities and pipelines throughout the United States, and display those in a way that is digestible to the user; be they an environmental scientist, legal professional, or curious citizen.
+
+## Challenges
+
+The existing [Oil & Gas Watch project](https://oilandgaswatch.org/) platform presented several challenges:
+
+- **Avoiding database management:** A The Savas Labs team wanted to avoid the complexities of managing a database infrastructure themselves.
+- **API accessibility:** A They needed a solution that could easily provide an API layer over the existing Postgres database without the need for extensive backend development.
+- **Data migration:** A Importing an existing Postgres database into a new system was a critical requirement.
+- **Geospatial data management:** A Savas Labs needed a solution to efficiently handle geospatial data with robust querying capabilities and integration with mapping tools. In addition, handling [Postgres branching](https://xata.io/branching) for different project environments posed unique challenges, which required Xata’s support.
+- **Collaborative visibility:** A The team required a platform that enabled visibility into data for better collaboration and project management.
+
+## Solution
+
+After evaluating several Database-as-a-Service (DBaaS) solutions, Savas Labs chose Xata for its simplicity and effectiveness in meeting their specific needs.
+
+![Building with Xata](/images/savas-xata-table.png)
+
+**Key factors for choosing Xata:**
+
+- **Simplicity and speed:** Xata's ability to load data quickly and provide an immediate API endpoint was a significant advantage. The team could focus on data presentation rather than management , effectively handling tasks such as [Postgres schema migrations](https://xata.io/schema-migrations) without extensive manual intervention.
```

Comment:
```suggestion
- **Simplicity and speed:** Xata's ability to load data quickly and provide an immediate API endpoint was a significant advantage. The team could focus on data presentation rather than management, effectively handling tasks such as [Postgres schema migrations](https://xata.io/schema-migrations) without extensive manual intervention.
```